timelock/project/contracts/timelock-contract/src/utils.sw

```diff
@@ -1,7 +1,28 @@
 library;
 
 use ::data_structures::Asset;
-use std::{bytes::Bytes, hash::sha256};
+use std::{bytes::Bytes, hash::{Hash, Hasher, sha256}};
+
+impl Hash for Option<Bytes> {
+    fn hash(self, ref mut state: Hasher) {
+        match self {
+            Some(bytes) => bytes.hash(state),
+            None => 0.hash(state),
```

Comment:
Unsure about the `0`s here.
I assume that hashing 0 is the equivalent of None somehow and that it's also akin to multiplying by 0. If that is the case then lets save on costs by returning ZERO_B256? Or does that not work?

Reply:
Secondly, We ought to have this implementation in the std in the future

Reply:
The hash of 0 is not 0